### DISCUSSION TITLE
23.10.31 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#50] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 외부 API를 의존하는 OAuth 테스트 작성법 구글링
    - #41 에 레퍼런스를 등록해 두었습니다.
- 내일은 어떤 일을 하나요?
    - 일정이 너무 딜레이 되는 것 같아, 친구 기능부터 개발하려 합니다!

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트와 노래 매핑 + 테스트 코드 작성
- 내일은 어떤 일을 하나요?
    - 플레이리스트와 노래 매핑 + 테스트 코드 작성 continued..
    - 노래 : 스포티파이 API와 연동해서 데이터 가져오기
    - User와 플레이리스트 매핑
    - 우재님 코드 리뷰

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 비동기 Issue, PR 작성

- 내일은 어떤 일을 하나요?
  - 비동기를 테스트 하기 위한 다른 방법들 찾기
  - TempUser 를 기존 User로 대체
